src/styles/components/_details-page.scss

```diff
@@ -15,6 +15,12 @@
     width: 100%;
     margin: auto;
     background-color: map.get($colors, "accent-teal");
+
+    &:last-child {
+      @media (prefers-color-scheme: dark) {
+        background-color: var(--background-color);
+      }
+    }
```

Comment:
Is this doing anything? I think this is already handled by `.details-page__secondary--related-components`.

Reply:
Yes, it fixes a problem with the green bg appearing to the sides of the component inventory in dark mode.

Without that line of code:
![Screen Shot 2022-11-18 at 4 32 45 PM](https://user-images.githubusercontent.com/55593075/202806837-1d7c4c8f-e77c-47f3-9d85-85fa48883810.png)
vs with:
![image](https://user-images.githubusercontent.com/55593075/202807064-e5b0995d-24ae-44e5-a57b-751b86d3119b.png)

Reply:
Oh I see, gotta get to those really wide screen sizes! Okay, all good then!